app/src/components/formdemo/FormDemo.tsx

```diff
@@ -750,6 +752,148 @@ export default function FormDemo() {
 		),
 	});
 
+	const tableProps = {
+		className: 'table',
+		headerRow: {
+			cells: [
+				{
+					renderedContent: <p>Observation Type</p>,
+					content: 'Observation Type',
+					align: 'left',
+				},
+				{
+					renderedContent: <p>Description</p>,
+					content: 'Description',
+					align: 'left',
+				},
+				{
+					renderedContent: <p>Example Use Case(s)</p>,
+					content: 'Example Use Case(s)',
+					align: 'left',
+				},
+			],
+		},
+		indentation: 0,
+		rows: [
+			{
+				raw: '|[Flow](flows/)|Returns a single metric showing currently observed flow interactions. |**Note**: There is no additional documentation for the flow observations.  The link will take you to the endpoints page for analytics. |\n',
```

Comment:
`DataTableRow` does not have a `raw` property

Reply:
DataTableCell has a raw property and DataTableRow is an array of DataTableCell